packages/nimble-components/src/table/components/row/styles.ts

```diff
@@ -29,6 +29,10 @@ export const styles = css`
         padding-left: 0px;
     }
 
+    .row-front-spacer {
+        min-width: ${controlHeight};
```

Comment:
Why `min-width` instead of `width`?

Reply:
I used that to prevent it from shrinking, but that was improper. I've switched to using flex settings.